### ISSUE TITLE
Component is not re-rendering

### ISSUE THREAD

**Issue by waifo** [maheshsenni/react-treelist#8]
On changing the data the component does not re-render , i tried changing the state in treelist component but it still does not re-render

**Comment by maheshsenni** [comment]
To render large data sets, I had to do a few optimizations. The code that needs to be updated is present [here](https://github.com/maheshsenni/react-treelist/blob/master/src/js/Body.js#L134). It ignores changes to `data` as of now. I'll try to take a look at it sometime this week.

```js
shouldComponentUpdate(nextProps, nextState) {
    // adding this as scroll causes re-render
    // checking just expanded rows for now, can add
    // 'data' in future
    return nextState.scrollTop !== this.state.scrollTop ||
      nextState.scrollLeft !== this.state.scrollLeft ||
      nextProps.width !== this.props.width ||
      nextState.expandedRows.length !== this.state.expandedRows.length ||
      nextProps.updateHash.sort !== this.props.updateHash.sort;
  }

**Comment by venkateshreddy** [comment]
Hi Mahesh,
Have you fixed this issue. We are also facing this issue.

**Comment by maheshsenni** [comment]
I have a fix ready and will publish this today to npm.

**Comment by maheshsenni** [comment]
This is fixed in v0.3.2. I have published this to npm.
